BEIMA.Backend.FT/DeviceFT.cs

```diff
@@ -76,5 +79,98 @@ public async Task DeviceNotInDatabase_AddDevice_CreatesNewDevice()
             Assert.That(getDevice.Location?.Longitude, Is.EqualTo(device.Location.Longitude));
             Assert.That(getDevice.Location?.Latitude, Is.EqualTo(device.Location.Latitude));
         }
+
+        [Test, Explicit("Must have a cleared database.")]
+        public async Task DevicesInDatabase_GetDeviceList_ReturnsDeviceList()
+        {
+            // ARRANGE
+            var deviceList = new List<Device>
+            {
+                new Device
+                {
+                    DeviceTag = "A-2",
+                    DeviceTypeId = "192830495728394823103456",
+                    Location = new Location
+                    {
+                        BuildingId = "192830495728394829103986",
+                        Latitude = "101.001",
+                        Longitude = "6.234",
+                        Notes = "Outside",
+                    },
+                    Manufacturer = "Generic Inc.",
+                    ModelNum = "44tsec",
+                    Notes = "Giberish",
+                    SerialNum = "tt4s",
+                    YearManufactured = 2010,
+                },
+                new Device
+                {
+                    DeviceTag = "B-1",
+                    DeviceTypeId = "902830495728394829103456",
+                    Location = new Location
+                    {
+                        BuildingId = "192831695728394829103456",
+                        Latitude = "74.003",
+                        Longitude = "138.123",
+                        Notes = "Inside",
+                    },
+                    Manufacturer = "Generic Labs",
+                    ModelNum = "dbbr6f",
+                    Notes = "Blah Blah",
+                    SerialNum = "c4ta",
+                    YearManufactured = 2011,
+                },
+                new Device
+                {
+                    DeviceTag = "C-3",
+                    DeviceTypeId = "192831195728394829103456",
+                    Location = new Location
+                    {
+                        BuildingId = "192830495728214829103456",
+                        Latitude = "101.989",
+                        Longitude = "25.004",
+                        Notes = "Above",
+                    },
+                    Manufacturer = "Generic Company",
+                    ModelNum = "y5eyf",
+                    Notes = "qwerty",
+                    SerialNum = "t4vw",
+                    YearManufactured = 2012,
+                },
```

Comment:
Should we be inserting these into a test database instead of the actual devices database? I used Postman to hit this endpoint and I noticed it returned these same devices here

Reply:
The issue is that the functional tests are running externally from the backend application itself. So because the backend application uses the actual database when we start it up on the backend, the functional tests have to use that same one. 

This particular test is marked as explicit, which means it isn't going to run on the build machine, and can only be run manually.

Reply:
Oh I see, sounds good